conformance_tests/sysman/test_sysman_device/src/test_sysman_device.cpp

```diff
@@ -165,6 +169,31 @@ TEST_F(
   }
 }
 
+#ifdef USE_ZESINIT
+TEST_F(
+    SYSMAN_DEVICE_TEST,
+    GivenValidDeviceWhenRetrievingSubDevicesThenEnsureNoSubDeviceUUIDMatchesDeviceUUID) {
+  for (auto device : devices) {
+    auto device_properties = lzt::get_sysman_device_properties(device);
+    uint32_t sub_devices_count = device_properties.numSubdevices;
+    if (sub_devices_count > 0) {
+      uint32_t num_sub_devices = 0;
+      auto sub_device_properties =
+          lzt::get_sysman_subdevice_properties(device, num_sub_devices);
+      EXPECT_EQ(sub_devices_count, num_sub_devices);
+      for (uint32_t sub_device_index = 0; sub_device_index < num_sub_devices;
+           sub_device_index++) {
+        EXPECT_LT(sub_device_properties[sub_device_index].subdeviceId,
```

Comment:
minor: Looking at the test name, it may not be necessary to validate this

Reply:
Resolved by recent commit. Thank you.